packages/@aws-cdk/aws-ecs/lib/ec2/ec2-service.ts

```diff
@@ -142,6 +145,36 @@ export class Ec2Service extends BaseService implements IEc2Service, elb.ILoadBal
     }
   }
 
+  /**
+   * Adds AWS X-Ray daemon as a sidecar container to enable tracing
+   */
+  public addTracing(props: TracingOptions = {}) {
+    // TODO: adjust task size based on container-level resources (cpu/memory)?
```

Comment:
issue?

Reply:
Applies to this issue: https://github.com/awslabs/aws-cdk/issues/2520